data/successors/successor outfits.txt

```diff
@@ -0,0 +1,635 @@
+	"afterburner energy" -1.2
+	"reverse flare sprite" "effect/successor flare/medium"
+		"frame rate" 20
+	"reverse flare sound" "ion medium"
+	description "This large reverse thruster sees use in both commerce and warfare, speeding up the traversal of trade routes in the former situation and enhancing combat agility in the latter."
+	description "	Because a reverse thruster must be facing forwards, it must be installed in the weapon section of your ship instead of the engine section."
+
+outfit `"Veusa" Inductive Thruster`
+	category "Engines"
+	series "Engines: Successors"
+	index 41
+	"cost" 3550000
+	thumbnail "outfit/large successor thruster"
+	licenses
+		"Successor"
+	"mass" 65
+	"outfit space" -65
+	"engine capacity" -65
+	"thrust" 78.0
+	"thrusting energy" 4.6
+	"thrusting heat" 3.0
+	"afterburner thrust" 117.0
+	"afterburner shields" 11.0
+	"afterburner heat" 7.5
+	"afterburner energy" 4.6
+	"afterburner effect" "large successor afterburner"
+	"flare sprite" "effect/successor flare/large"
+		"frame rate" 20
+	"flare sound" "ion large"
+	description `The largest of the Successors' pulsed inductive thrusters, the "Veusa" is used primarily to propel their large warships and freighters. While its high-performance mode is powerful enough to accelerate even the largest vessels to high speeds, it also drains shield power remarkably quickly; this is clearly meant to be paired with one of the Successors' fast-recharging shield generators.`
+
+outfit `"Veusa" Inductive Steering`
+	category "Engines"
+	series "Engines: Successors"
+	index 40
+	"cost" 2925000
+	thumbnail "outfit/large successor steering"
+	licenses
+		"Successor"
+	"mass" 42
+	"outfit space" -42
+	"engine capacity" -42
+	"afterburner energy" -2.4
+	"turn" 2268
+	"turning energy" 3.8
+	"turning heat" 3.1
+	"steering flare sprite" "effect/successor flare/large"
+		"frame rate" 20
+	"steering flare sound" "ion large"
+	description `The "Veusa" series of Successor steering units is significantly smaller than comparable human engines, allowing the Successors to gain some flexibility in the engine layouts of their large ships.`
+
+outfit `"Veusa" Inductive Reverser`
+	category "Engines"
+	series "Engines: Successors"
+	index 42
+	"cost" 3050000
+	thumbnail "outfit/large successor reverse"
+	licenses
+		"Successor"
+	"mass" 35
+	"outfit space" -35
+	"weapon capacity" -35
+	"afterburner energy" -2.1
+	"reverse thrust" 54.75
+	"reverse thrusting energy" 3.7
+	"reverse thrusting heat" 3.0
+	"reverse flare sprite" "effect/successor flare/large"
+		"frame rate" 20
+	"reverse flare sound" "ion large"
+	description "While the invertebrate Successors are relatively unaffected, human pilots must take care when using a reverse thruster this large to avoid dangerous g-force stress on the body."
+	description "	Because a reverse thruster must be facing forwards, it must be installed in the weapon section of your ship instead of the engine section."
+
+
+outfit "Multimodal Armor: Heavy"
```

Comment:
On the Aaulqra, this effectively increases the hull by 11000, putting its total HP up to 76000 - still slightly below that of the T2 Model 128. This also makes regenerated hull 20% more valuable, though the Regen multimodal armour is stronger in that specific role.
This outfit is limited to Successor ships, and doesn't make those ships overpowered, so I'd say it's reasonable.